documentation/book/proc-renewing-ca-certs-manually.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-renewing-ca-certs-manually-{context}']
+
+= Renewing CA certificates manually
+
+If automatic CA certificate generation is enabled (`generateCertificateAuthority=true`), the cluster CA and clients CA certificates will auto-renew at the start of their respective certificate renewal periods. You can manually renew one or both of these certificates before the certificate renewal period begins, if required for security reasons. A renewed certificate uses the same private key as the old certificate.
```

Comment:
I think this might confuse people since this is indeed `generateCertificateAuthority=true`, but it is also the default. So even without this being in their Kafka CR, they are still using automatic CA generation. We should probably clarify that. Also, there is a separate value for clients and cluster CAs. Do we have some chapter in the docs to link to for clarification and more details?

Reply:
Agree, adding that when we talk about a property in a CR we should describe the entire "path" so `Kafka.spec. ....`

Reply:
Good feedback. I'll update the procedure intro and provide some relevant links. Please re-review after I push the changes.